src/plugins/dashboard/public/application/embeddable/empty_screen/__snapshots__/dashboard_empty_screen.test.tsx.snap

```diff
@@ -160,14 +160,12 @@ exports[`DashboardEmptyScreen renders correctly with edit mode 1`] = `
       },
       "get$": [MockFunction],
       "getAll": [MockFunction],
-      "getSaved$": [MockFunction],
```

Comment:
Please, consider fixing the test. It shouldn't fail whenever the Core team updates any of the public contract methods. It creates a lot of false-negative failures.

Reply:
I've been meaning to fix these tests for sure. They don't need to be so restrictive.